src/registries/npm-registry.js

```diff
@@ -136,6 +136,10 @@ export default class NpmRegistry extends Registry {
   }
 
   getRegistry(packageName: string): string {
+    if (process.env.NPM_CONFIG_REGISTRY) {
```

Comment:
Would have to check for null as well in that case

Would a typeof === 'string' check be better?